src/lib/plot.typ

```diff
@@ -96,6 +103,12 @@
 ///                    `x-min: 0`.
 ///                    - min (int): Axis minimum
 ///                    - max (int): Axis maximum
+///                    - horizontal (bool): Axis orientation; note that each
+///                      plot must use one vertical and one horizontal axis!
+///                      The default value for this parameter is guessed: Axes
+///                      starting with "x" are considered horizontal by default.
+///                      This does not affect the side the ticks of the axis are
+///                      drawn, but only the drawing direction.
```

Comment:
I don't think this option is the best way to specify which axis should be horizontal. Its the having to check they are mutually exculsive part that does it. I think it shouldn't be so reliant on the axis setting.

Is this redundent given you have swapped axes in `plot.add` for the tests? Why not just document that properly? If not a test for `horizontal` should be added.

Reply:
Since axes are created on demand and the orientation is not bound to their use, we need an option to specify the orientation. You can also use axes in any combination you want.
This is not redundant when using custom axes, e.g. `axes: ("abc", "my-axis2")`.